gensim/models/tfidfmodel.py

```diff
@@ -219,7 +222,8 @@ class TfidfModel(interfaces.TransformationABC):
     """
 
     def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.identity,
-                 wglobal=df2idf, normalize=True, smartirs=None):
+                 wglobal=df2idf, normalize=True, smartirs=None,
+                 pivot_norm=False, slope=0.65, pivot=100):
```

Comment:
I'm not sure about default values here, we really can't reduce it to 2 values (join `pivot_norm` and `pivot`)?